_includes/navigation.html

```diff
@@ -29,11 +29,11 @@
 					{% for sublink in link.subitems %}
 					<li >
 						<div {% if sublink.link == page.url %} class="sub-nav-li sub-nav-li--active" {% else %} class="sub-nav-li" {% endif %}>
-								<img class="nav-img" src="{{sublink.img_src}}">
+								<img class="nav-img" loading="lazy" src="{{sublink.img_src}}">
```

Comment:
I don't think we should not lazy load navigation because it is directly visible to user when the page loads

Reply:
@saisuvanth ??

Reply:
Okay I'll change that into normal loading 
any fixes other than that?

Reply:
@saisuvanth, resolve the conflicts please